src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculation, Draw {
+
+    public String getColor() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        return colorSupplier.getRandomColor();
+    }
```

Comment:
Color is a common field
```suggestion
    private String color;
    
    public Figure (String color) {
        this.color = color;
    }
    
    public String getColor() {
        return this.color;
    }
```